source/extensions/filters/common/expr/context.cc

```diff
@@ -129,6 +130,8 @@ absl::optional<CelValue> ResponseWrapper::operator[](CelValue key) const {
     return CelValue::CreateMap(&headers_);
   } else if (value == Trailers) {
     return CelValue::CreateMap(&trailers_);
+  } else if (value == Flags) {
+    return CelValue::CreateString(StreamInfo::ResponseFlagUtils::toShortString(info_));
```

Comment:
Unfortunately, this won't work unless we make `toShortString` return a `const std::string&`. The context does not store any state, so it has to capture the string by pointer. The constructor is dangerous as-is, so it was changed recently to prevent this sort of the problem.

Reply:
Response wrapper seems to be only reasonable owner of this string. If that is not acceptable, then pass in a integer and let module do string conversion?

Reply:
Or I guess just make a special WASM API for it, instead of using the generalized data accessor.

Reply:
Envoy could own the constant and pass it by reference. It'd take some refactoring, but probably minimal. You're raising an important point though, whether we want these flags surface in WASM API as well or we want something more generic. @mandarjog , thoughts?

Reply:
Updated it to return an integer.